### PR TITLE
CA-134 Deprecates s3 policy arns in EMR module

### PR DESCRIPTION
**Pull request by fvazquez-caylent**
v7.4.0 - January 7th, 2022
* Deprecates `s3_policy_arns` in favor of `additional_policy_arns`.

### CODE REVIEW

**Comment by tamr-teamcity** [comment]
This PR doesn't appear to be linked to a DevOps/SRE jira ticket

File: examples/static-hbase-logs/cloudwatch.tf [hunk 1]
```diff
@@ -0,0 +1,12 @@
+resource "aws_cloudwatch_log_group" "tamr_log_group" {
+  name = format("%s-%s", var.name_prefix, "tamr_log_group")
+  tags = var.tags
+}
+
+# Get current Region
+data "aws_region" "current" {}
+
+resource "local_file" "postfix_config" {
```
Comment (franco-caylent) [inline examples/static-hbase-logs/cloudwatch.tf:9]: Let's rename the file to something more meaningful

File: examples/static-hbase-logs/networking.tf [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+  # source               = "git::https://github.com/Datatamer/terraform-aws-emr.git//modules/aws-emr-ports?ref=6.2.0"
+  source       = "../../modules/aws-emr-ports"
+  applications = local.this_application
+}
+
+module "aws-emr-sg-master" {
+  source              = "git::git@github.com:Datatamer/terraform-aws-security-groups.git?ref=1.0.0"
+  vpc_id              = var.vpc_id
+  ingress_cidr_blocks = var.ingress_cidr_blocks
+  #ingress_security_groups = module.sg_vm_web.security_group_ids
```
Comment (franco-caylent) [inline examples/static-hbase-logs/networking.tf:11]: remove the commented line

File: examples/static-hbase-logs/emr.tf [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+  source     = "terraform-aws-modules/key-pair/aws"
+  version    = "1.0.0"
+  key_name   = "hbase-test-emr-key"
+  public_key = tls_private_key.emr_private_key.public_key_openssh
+  tags       = var.tags
+}
+
+# EMR Static HBase cluster
+module "emr-hbase" {
+  # source = "git::git@github.com:Datatamer/terraform-aws-emr.git?ref=6.2.0"
```
Comment (franco-caylent) [inline examples/static-hbase-logs/emr.tf:20]: You are going to need a dependency with the VPC Endpoint in order for this not to fail randomly. EMR takes longer to boot than Tamr-VM but i'd rather be safe than sorry.

File: examples/static-hbase-logs/networking.tf [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+  ingress_security_groups = [module.aws-emr-sg-master.security_group_ids[0],
+  module.aws-emr-sg-core.security_group_ids[0]]
+  sg_name_prefix   = format("%s-%s", var.name_prefix, "interface-endpoint-sg")
+  ingress_protocol = "tcp"
+  egress_protocol  = "all"
+  tags             = var.tags
+}
+
+#Create VPC Interface Endpoint for Cloudwatch
+module "endpoints" {
```
Comment (franco-caylent) [inline examples/static-hbase-logs/networking.tf:58]: We are going to migrate this to the networking module later right?

**Review by franco-caylent — COMMENTED** [review]
Looking good overall. I think you missed one though.
```
grep -r "s3_policy_arns"
./static-hbase/main.tf:  s3_policy_arns                 = [module.emr-logs-bucket.rw_policy_arn, module.emr-rootdir-bucket.rw_policy_arn]
```

File: examples/static-hbase-logs/networking.tf (hunk 4 above)
Comment (fvazquez-caylent) [reply examples/static-hbase-logs/networking.tf:58]: Yes. This is being worked right now.

**Comment by fvazquez-caylent** [comment]
Opening a new one.

**Pull request closed without merging**